src/defaultTheme.ts

```diff
@@ -89,6 +89,7 @@ const defaultTheme: Theme = {
   space: n => `${n * 16}px`,
   gap: n => `${n * 48}px`,
   block: n => `${n * 96}px`,
+  borderRadius: '0.1rem',
```

Comment:
I really like this addition. I think i'll update the rest of the repo to use this